_activities/gsoc.md

```diff
@@ -15,42 +15,58 @@ and analyze petabytes data from high-energy physics experiments, such as the Lar
 hosted at the CERN laboratory in Geneva, Switzerland. 
 Some of the questions that we collectively ask are: 
 
-- what are the fundamental blocks that make up our Universe? 
-- what is the nature of dark matter and dark energy?
-- what is the nature of the asymmetry between matter and antimatter? 
-- what was early Universe like? 
+- What are the fundamental blocks that make up our Universe?
+- What is the nature of dark matter and dark energy?
+- What is the nature of the asymmetry between matter and antimatter?
+- What was the early Universe like?
+
 
 To answer these questions, particle physicists build software to simulate and analyze what happens in particle physics detectors.
 
-The CERN software for experiments (CERN-SFT) group has participated in the GSoC since 2011. 
-Since 2017 the program has expanded to involve the high-energy physics community under the umbrella of the HEP Software Foundation.
+The CERN software for experiments (CERN-SFT) group has participated in the GSoC since 2011. Since 2017 the program has expanded to involve the entire high-energy physics community under the umbrella of the HEP Software Foundation.
+
+Information from last year’s GSoC can be found [here](/gsoc/2017/index.html). In 2018 CERN-HSF is again applying for participation in the program. 
 
-Information from last year's GSoC can be found [here](/gsoc/2017/index.html). For 2018 the 
-HSF is again applying for participation in the program. If you are 
-interested in the GSoC program contact us using the HSF GSoC mailing list: [hep-software-foundation-google-summer-of-code@googlegroups.com](mailto:hep-software-foundation-google-summer-of-code@googlegroups.com).
+If you are interested in the GSoC program contact us using the HSF GSoC mailing list: [hep-software-foundation-google-summer-of-code@googlegroups.com](mailto:hep-software-foundation-google-summer-of-code@googlegroups.com)
 
 Instructions for participating projects and mentors can be found [here](/gsoc/guideline.html).
 
-## Projects in 2018
+## 2018 Projects
```

Comment:
Now you introduced an asymmetry in the two titles for projects and institutes - we should do one or the other, but be consistent!